server/src/resolvers/__tests__/eventBookClub.js

```diff
@@ -0,0 +1,98 @@
+const {EventBookClub} = require('../../resolvers');
```

Comment:
File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation

Please check other file names too.